test/src/test/java/org/corfudb/integration/CorfuReplicationReconfigurationIT.java

```diff
@@ -203,7 +203,10 @@ public void testActiveClusterReset() throws Exception {
 
         // (9) Stop active LR again, so server restarts from Log Entry Sync, with no actual deltas (as there is no new data)
         // and verify remainingEntriesToSend is still '0'
+
         stopActiveLogReplicator();
+        Sleep.sleepUninterruptibly(Duration.ofSeconds(LogReplicationAckReader.ACKED_TS_READ_INTERVAL_SECONDS + delta));
```

Comment:
any reason for adding the sleep here?

Reply:
@Jas0nch  had added it probably to avoid some tests failing. Reverting now.